### ISSUE TITLE
Fails to write image data

### ISSUE THREAD

**Issue by melinath** [danifus/django_override_storage#3]
I have an image generated with factoryboy using the following method:

```
factory.django.ImageField()._make_data({'width': 100})
```

This is the image:

![image](https://user-images.githubusercontent.com/299979/40083541-5399a596-5849-11e8-831b-b88c60035eec.png)

When I try to save this file using the locmem storage backend, I get the following error:

```
File "override_storage/storage.py", line 31, in _save
    content = content.encode()
UnicodeDecodeError: 'ascii' codec can't decode byte 0xff in position 0: ordinal not in range(128)
```

I think the issue is that only unicode data should be converted to bytes - byte data should be left as-is.

**Comment by danifus** [comment]
Thanks for the report!

I missed this bug when adding Python 2 support - Python 3 bytes objects don't have the ``.encode`` method so I was catching the ``AttributeError``s raised based on that. I've made it more explicit and added a test for it.

Fixed in the 0.1.5 release, just pushed to PyPI.

Let me know if you still have problems. Thanks!

**Comment by melinath** [comment]
That issue seems to be resolved by 1f80d1ccc19bae770b7f98fb16fdd00dfb54e6e0 - thanks for the quick response!
